pkg/cmd/show.go

```diff
@@ -269,7 +266,7 @@ func showPrometheus(targetReader TargetReader) {
 func showAltermanager(targetReader TargetReader) {
```

Comment:
The code in here maybe lack of maintenance, I found the alertmanger is not working. 
result check from my side as below, is alermanger move somewhere else ? or this method only working in specific cluster
```
k get ingress -n shoot--i333878--aws-users-role -o name
ingress.extensions/dashboard-terminal-kube-apiserver
ingress.extensions/grafana-operators
ingress.extensions/grafana-users
ingress.extensions/prometheus
```

Reply:
ok I see, It seems from under -n garden. will it

Reply:
yes take care later if needed

Reply:
https://github.com/gardener/gardenctl/issues/350